app/controllers/provider_interface/application_data_export_controller.rb

```diff
@@ -21,6 +21,7 @@ def export
               :provider,
               :accredited_provider,
               :site,
+              :offered_course_option,
```

Comment:
do we need the same includes as the `course_option` here? `%i[course site]`

Reply:
Yup. Good shout